Microsoft.HealthVault.Fhir/FhirExtensions/FhirDateTimeExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Hl7.Fhir.Model;
+
+namespace Microsoft.HealthVault.Fhir.FhirExtensions
+{
+    public static class FhirDateTimeExtensions
+    {
+        public enum FhirDateTimePrecision
+        {
+            Year = 4,       //2017
+            Month = 7,      //2017-22
+            Day = 10,       //2017-22-21
+            Minute = 15,    //2017-22-21T13:45
+            Second = 18    //2017-22-21T13:45:21
+        }
```

Comment:
This looks really cryptic, please add comments on what this enum's purpose is. Are you using an enum to count characters?